ILRepack/ILRepack.cs

```diff
@@ -352,7 +352,9 @@ public void Repack()
                 Logger.Info("Output directory does not exist. Creating output directory: " + outputDir);
                 Directory.CreateDirectory(outputDir);
             }
-            TargetAssemblyDefinition.Write(Options.OutputFile, parameters);
+            using (FileStream outputStream = File.OpenWrite(Options.OutputFile))
+                TargetAssemblyDefinition.Write(outputStream, parameters);
```

Comment:
I mean `AssemblyDefinition` one is only doing `main_module.Write (fileName, parameters);` and main_module is a ModuleDefinition, correct ?